tests/unit/test_exceptions.py

```diff
@@ -4,6 +4,9 @@
     ClientException,
     PRAWException,
```

Comment:
It looks like these weren't in the right order to begin with. Can we fix that while we're here?

Reply:
I'll pull a new isort pr to fix this

Reply:
Nvm, they can be fixed here